src/services/insights/services/avgPageLoaded.js

```diff
@@ -0,0 +1,68 @@
+const request = require('request-promise-native');
+const hooks = require('../hooks');
+const { URL } = require('url');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`url` import should occur before import of `../hooks`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4384612)